### PR TITLE
version: Update kernel version to 4.9.24-60

### LINKED ISSUE #850: docker: Error response from daemon: shim error: open pid: no such file or directory on Centos
It is not possible to run cc on centos 7.0, as I have the following error
$ docker run -ti fedora bash
docker: Error response from daemon: shim error: open pid: no such file or directory.
Currently I am using the following
clear-containers.img -> clear-14700-containers.img
vmlinux.container -> vmlinux-4.9.4-58.container
Docker version 17.04.0-ce, build 4845c56
cc-oci-runtime version: 2.1.5
spec version: 1.0.0-rc1
commit: 4561b005f009d34a7dec28b1c08b72124ab8d0a6

Here it is the log of cc-proxy
$ ./cc-proxy -v 3
I0426 09:05:29.584890   21372 proxy.go:328] listening on /var/run/cc-oci-runtime/proxy.sock
I0426 09:05:29.584933   21372 proxy.go:370] proxy started
I0426 09:05:39.641799   21372 proxy.go:67] [client #1] client connected
I0426 09:05:39.709354   21372 proxy.go:77] [client #1] hello(containerId=ec005602a273d67601d0249eef5e1b04641b50f7ec17111f6491d58c096c6831,ctlSerial=/var/run/cc-oci-runtime/ec005602a273d67601d0249eef5e1b04641b50f7ec17111f6491d58c096c6831/ga-ctl.sock,ioSerial=/var/run/cc-oci-runtime/ec005602a273d67601d0249eef5e1b04641b50f7ec17111f6491d58c096c6831/ga-tty.sock,console=/var/run/cc-oci-runtime/ec005602a273d67601d0249eef5e1b04641b50f7ec17111f6491d58c096c6831/console.sock)
I0426 09:05:39.888701   21372 vm.go:114] [vm ec005602 hyperstart] open hyper channel /dev/vport1p1
I0426 09:05:39.888826   21372 vm.go:114] [vm ec005602 hyperstart] send ready message
I0426 09:05:39.888843   21372 vm.go:114] [vm ec005602 hyperstart] hyper send type 8, len 0
I0426 09:05:39.888995   21372 vm.go:114] [vm ec005602 hyperstart] open hyper channel /dev/vport1p2
I0426 09:05:39.889099   21372 vm.go:114] [vm ec005602 hyperstart] hyper_init_event hyper channel event 0x4201f0, ops 0x420030, fd 3
I0426 09:05:39.889211   21372 vm.go:114] [vm ec005602 hyperstart] hyper_add_event add event fd 3, 0x420030
I0426 09:05:39.889277   21372 vm.go:114] [vm ec005602 hyperstart] hyper_init_event hyper ttyfd event 0x4201b8, ops 0x420010, fd 4
I0426 09:05:39.889417   21372 proxy.go:77] [client #1] hyper(cmd=startpod, data={"hostname":"ec005602a273","containers":[],"shareDir":"rootfs","interfaces":[{"macAddr":"02:42:ac:11:00:02","newDeviceName":"eth0","mtu":1500,"ipAddresses":[{"ipAddress":"172.17.0.2","netMask":"255.255.0.0"}]}],"routes":[{"dest":"default","gateway":"172.17.0.1","device":"eth0"},{"dest":"172.17.0.0","device":"eth0"}],"iptableRules":"# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*nat\n:PREROUTING ACCEPT [0:0]\n:INPUT ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\n:POSTROUTING ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*mangle\n:PREROUTING ACCEPT [0:0]\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\n:POSTROUTING ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*security\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*raw\n:PREROUTING ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*filter\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n"})
I0426 09:05:39.889535   21372 vm.go:114] [vm ec005602 hyperstart] hyper_add_event add event fd 4, 0x420010
I0426 09:05:39.889613   21372 vm.go:114] [vm ec005602 hyperstart] hyper_loop epoll_wait 1
I0426 09:05:39.889688   21372 vm.go:114] [vm ec005602 hyperstart] hyper_handle_event get event 1, he 0x4201f0, fd 3. ops 0x420030
I0426 09:05:39.889752   21372 vm.go:114] [vm ec005602 hyperstart] hyper_handle_event event EPOLLIN, he 0x4201f0, fd 3, 0x420030
I0426 09:05:39.889830   21372 vm.go:114] [vm ec005602 hyperstart] hyper_channel_read
I0426 09:05:39.889893   21372 vm.go:114] [vm ec005602 hyperstart] hyper send type 14, len 4
I0426 09:05:39.889990   21372 vm.go:114] [vm ec005602 hyperstart] get length 1355
I0426 09:05:39.890057   21372 vm.go:114] [vm ec005602 hyperstart] hyper send type 14, len 4
I0426 09:05:39.890273   21372 vm.go:114] [vm ec005602 hyperstart] 0 0 0 1 0 0 5 4b 7b 22 68 6f 73 74 6e 61 6d 65 22 3a 22 65 63 30 30 35 36 30 32 61 32 37 33 22 2c 22 63 6f 6e 74 61 69 6e 65 72 73 22 3a 5b 5d 2c 22 73 68 61 72 65 44 69 72 22 3a 22 72 6f 6f 74 66 73 22 2c 22 69 6e 74 65 72 66 61 63 65 73 22 3a 5b 7b 22 6d 61 63 41 64 64 72 22 3a 22 30 32 3a 34 32 3a 61 63 3a 31 31 3a 30 30 3a 30 32 22 2c 22 6e 65 77 44 65 76 69 63 65 4e 61 6d 65 22 3a 22 65 74 68 30 22 2c 22 6d 74 75 22 3a 31 35 30 30 2c 22 69 70 41 64 64 72 65 73 73 65 73 22 3a 5b 7b 22 69 70 41 64 64 72 65 73 73 22 3a 22 31 37 32 2e 31 37 2e 30 2e 32 22 2c 22 6e 65 74 4d 61 73 6b 22 3a 22 32 35 35 2e 32 35 35 2e 30 2e 30 22 7d 5d 7d 5d 2c 22 72 6f 75 74 65 73 22 3a 5b 7b 22 64 65 73 74 22 3a 22 64 65 66 61 75 6c 74 22 2c 22 67 61 74 65 77 61 79 22 3a 22 31 37 32 2e 31 37 2e 30 2e 31 22 2c 22 64 65 76 69 63 65 22 3a 22 65 74 68 30 22 7d 2c 7b 22 64 65 73 74 22 3a 22 31 37 32 2e 31 37 2e 30 2e 30 22 2c 22 64 65 76 69 63 65 22 3a 22 65 74 68 30 22 7d 5d 2c 22 69 70 74 61 62 6c 65 52 75 6c 65 73 22 3a 22 23 20 47 65 6e 65 72 61 74 65 64 20 62 79 20 69 70 74 61 62 6c 65 73 2d 73 61 76 65 20 76 31 2e 34 2e 32 31 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 2a 6e 61 74 5c 6e 3a 50 52 45 52 4f 55 54 49 4e 47 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 49 4e 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 4f 55 54 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 50 4f 53 54 52 4f 55 54 49 4e 47 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 43 4f 4d 4d 49 54 5c 6e 23 20 43 6f 6d 70 6c 65 74 65 64 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 23 20 47 65 6e 65 72 61 74 65 64 20 62 79 20 69 70 74 61 62 6c 65 73 2d 73 61 76 65 20 76 31 2e 34 2e 32 31 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 2a 6d 61 6e 67 6c 65 5c 6e 3a 50 52 45 52 4f 55 54 49 4e 47 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 49 4e 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 46 4f 52 57 41 52 44 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 4f 55 54 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 50 4f 53 54 52 4f 55 54 49 4e 47 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 43 4f 4d 4d 49 54 5c 6e 23 20 43 6f 6d 70 6c 65 74 65 64 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 23 20 47 65 6e 65 72 61 74 65 64 20 62 79 20 69 70 74 61 62 6c 65 73 2d 73 61 76 65 20 76 31 2e 34 2e 32 31 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 2a 73 65 63 75 72 69 74 79 5c 6e 3a 49 4e 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 46 4f 52 57 41 52 44 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 4f 55 54 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 43 4f 4d 4d 49 54 5c 6e 23 20 43 6f 6d 70 6c 65 74 65 64 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 23 20 47 65 6e 65 72 61 74 65 64 20 62 79 20 69 70 74 61 62 6c 65 73 2d 73 61 76 65 20 76 31 2e 34 2e 32 31 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 2a 72 61 77 5c 6e 3a 50 52 45 52 4f 55 54 49 4e 47 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 4f 55 54 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 43 4f 4d 4d 49 54 5c 6e 23 20 43 6f 6d 70 6c 65 74 65 64 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 23 20 47 65 6e 65 72 61 74 65 64 20 62 79 20 69 70 74 61 62 6c 65 73 2d 73 61 76 65 20 76 31 2e 34 2e 32 31 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 2a 66 69 6c 74 65 72 5c 6e 3a 49 4e 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 46 4f 52 57 41 52 44 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 3a 4f 55 54 50 55 54 20 41 43 43 45 50 54 20 5b 30 3a 30 5d 5c 6e 43 4f 4d 4d 49 54 5c 6e 23 20 43 6f 6d 70 6c 65 74 65 64 20 6f 6e 20 57 65 64 20 41 70 72 20 32 36 20 30 39 3a 30 35 3a 33 39 20 32 30 31 37 5c 6e 22 7d
I0426 09:05:39.890367   21372 vm.go:114] [vm ec005602 hyperstart]  hyper_channel_handle, type 1, len 1355
I0426 09:05:39.890444   21372 vm.go:114] [vm ec005602 hyperstart] call hyper_start_pod, json {"hostname":"ec005602a273","containers":[],"shareDir":"rootfs","interfaces":[{"macAddr":"02:42:ac:11:00:02","newDeviceName":"eth0","mtu":1500,"ipAddresses":[{"ipAddress":"172.17.0.2","netMask":"255.255.0.0"}]}],"routes":[{"dest":"default","gateway":"172.17.0.1","device":"eth0"},{"dest":"172.17.0.0","device":"eth0"}],"iptableRules":"# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*nat\n:PREROUTING ACCEPT [0:0]\n:INPUT ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\n:POSTROUTING ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*mangle\n:PREROUTING ACCEPT [0:0]\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\n:POSTROUTING ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*security\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*raw\n:PREROUTING ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*filter\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n"}, len 1347
I0426 09:05:39.890599   21372 vm.go:114] [vm ec005602 hyperstart] call hyper_start_pod, json {"hostname":"ec005602a273","containers":[],"shareDir":"rootfs","interfaces":[{"macAddr":"02:42:ac:11:00:02","newDeviceName":"eth0","mtu":1500,"ipAddresses":[{"ipAddress":"172.17.0.2","netMask":"255.255.0.0"}]}],"routes":[{"dest":"default","gateway":"172.17.0.1","device":"eth0"},{"dest":"172.17.0.0","device":"eth0"}],"iptableRules":"# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*nat\n:PREROUTING ACCEPT [0:0]\n:INPUT ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\n:POSTROUTING ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*mangle\n:PREROUTING ACCEPT [0:0]\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\n:POSTROUTING ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*security\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*raw\n:PREROUTING ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017\n*filter\n:INPUT ACCEPT [0:0]\n:FORWARD ACCEPT [0:0]\n:OUTPUT ACCEPT [0:0]\nCOMMIT\n# Completed on Wed Apr 26 09:05:39 2017\n"}, len 1347
I0426 09:05:39.890707   21372 vm.go:114] [vm ec005602 hyperstart] jsmn parse successed, n is 39
I0426 09:05:39.890781   21372 vm.go:114] [vm ec005602 hyperstart] token 0, type is 1, size is 6
I0426 09:05:39.890840   21372 vm.go:114] [vm ec005602 hyperstart] token 1, type is 3, size is 1
I0426 09:05:39.890922   21372 vm.go:114] [vm ec005602 hyperstart] hostname is ec005602a273
I0426 09:05:39.890982   21372 vm.go:114] [vm ec005602 hyperstart] token 3, type is 3, size is 1
I0426 09:05:39.891047   21372 vm.go:114] [vm ec005602 hyperstart] container count 0
I0426 09:05:39.891115   21372 vm.go:114] [vm ec005602 hyperstart] token 5, type is 3, size is 1
I0426 09:05:39.891176   21372 vm.go:114] [vm ec005602 hyperstart] share tag is rootfs
I0426 09:05:39.891235   21372 vm.go:114] [vm ec005602 hyperstart] token 7, type is 3, size is 1
I0426 09:05:39.891296   21372 vm.go:114] [vm ec005602 hyperstart] network interfaces num 1
I0426 09:05:39.891360   21372 vm.go:114] [vm ec005602 hyperstart] net mac address is 02:42:ac:11:00:02
I0426 09:05:39.891424   21372 vm.go:114] [vm ec005602 hyperstart] new interface name is eth0
I0426 09:05:39.891490   21372 vm.go:114] [vm ec005602 hyperstart] mtu is 1500
I0426 09:05:39.891568   21372 vm.go:114] [vm ec005602 hyperstart] Number of ip addresses:1
I0426 09:05:39.891645   21372 vm.go:114] [vm ec005602 hyperstart] net ipaddress for device (null) is 172.17.0.2
I0426 09:05:39.891714   21372 vm.go:114] [vm ec005602 hyperstart] net mask for device (null) is 255.255.0.0
I0426 09:05:39.891782   21372 vm.go:114] [vm ec005602 hyperstart] token 23, type is 3, size is 1
I0426 09:05:39.891850   21372 vm.go:114] [vm ec005602 hyperstart] network routes num 2
I0426 09:05:39.891916   21372 vm.go:114] [vm ec005602 hyperstart] route 0 dest is default
I0426 09:05:39.891981   21372 vm.go:114] [vm ec005602 hyperstart] route 0 gateway is 172.17.0.1
I0426 09:05:39.892048   21372 vm.go:114] [vm ec005602 hyperstart] route 0 device is eth0
I0426 09:05:39.892113   21372 vm.go:114] [vm ec005602 hyperstart] route 1 dest is 172.17.0.0
I0426 09:05:39.892179   21372 vm.go:114] [vm ec005602 hyperstart] route 1 device is eth0
I0426 09:05:39.892243   21372 vm.go:114] [vm ec005602 hyperstart] token 37, type is 3, size is 1
I0426 09:05:39.892312   21372 vm.go:114] [vm ec005602 hyperstart] iptable rules provided:# Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017
I0426 09:05:39.892381   21372 vm.go:114] [vm ec005602 hyperstart] *nat
I0426 09:05:39.892446   21372 vm.go:114] [vm ec005602 hyperstart] :PREROUTING ACCEPT [0:0]
I0426 09:05:39.892512   21372 vm.go:114] [vm ec005602 hyperstart] :INPUT ACCEPT [0:0]
I0426 09:05:39.892577   21372 vm.go:114] [vm ec005602 hyperstart] :OUTPUT ACCEPT [0:0]
I0426 09:05:39.892681   21372 vm.go:114] [vm ec005602 hyperstart] :POSTROUTING ACCEPT [0:0]
I0426 09:05:39.892743   21372 vm.go:114] [vm ec005602 hyperstart] COMMIT
I0426 09:05:39.892806   21372 vm.go:114] [vm ec005602 hyperstart] # Completed on Wed Apr 26 09:05:39 2017
I0426 09:05:39.892868   21372 vm.go:114] [vm ec005602 hyperstart] # Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017
I0426 09:05:39.892930   21372 vm.go:114] [vm ec005602 hyperstart] *mangle
I0426 09:05:39.892992   21372 vm.go:114] [vm ec005602 hyperstart] :PREROUTING ACCEPT [0:0]
I0426 09:05:39.893059   21372 vm.go:114] [vm ec005602 hyperstart] :INPUT ACCEPT [0:0]
I0426 09:05:39.893124   21372 vm.go:114] [vm ec005602 hyperstart] :FORWARD ACCEPT [0:0]
I0426 09:05:39.893193   21372 vm.go:114] [vm ec005602 hyperstart] :OUTPUT ACCEPT [0:0]
I0426 09:05:39.893246   21372 vm.go:114] [vm ec005602 hyperstart] :POSTROUTING ACCEPT [0:0]
I0426 09:05:39.893298   21372 vm.go:114] [vm ec005602 hyperstart] COMMIT
I0426 09:05:39.893348   21372 vm.go:114] [vm ec005602 hyperstart] # Completed on Wed Apr 26 09:05:39 2017
I0426 09:05:39.893399   21372 vm.go:114] [vm ec005602 hyperstart] # Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017
I0426 09:05:39.893450   21372 vm.go:114] [vm ec005602 hyperstart] *security
I0426 09:05:39.893512   21372 vm.go:114] [vm ec005602 hyperstart] :INPUT ACCEPT [0:0]
I0426 09:05:39.893565   21372 vm.go:114] [vm ec005602 hyperstart] :FORWARD ACCEPT [0:0]
I0426 09:05:39.893615   21372 vm.go:114] [vm ec005602 hyperstart] :OUTPUT ACCEPT [0:0]
I0426 09:05:39.893673   21372 vm.go:114] [vm ec005602 hyperstart] COMMIT
I0426 09:05:39.893722   21372 vm.go:114] [vm ec005602 hyperstart] # Completed on Wed Apr 26 09:05:39 2017
I0426 09:05:39.893773   21372 vm.go:114] [vm ec005602 hyperstart] # Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017
I0426 09:05:39.893823   21372 vm.go:114] [vm ec005602 hyperstart] *raw
I0426 09:05:39.893872   21372 vm.go:114] [vm ec005602 hyperstart] :PREROUTING ACCEPT [0:0]
I0426 09:05:39.893923   21372 vm.go:114] [vm ec005602 hyperstart] :OUTPUT ACCEPT [0:0]
I0426 09:05:39.893973   21372 vm.go:114] [vm ec005602 hyperstart] COMMIT
I0426 09:05:39.894023   21372 vm.go:114] [vm ec005602 hyperstart] # Completed on Wed Apr 26 09:05:39 2017
I0426 09:05:39.894073   21372 vm.go:114] [vm ec005602 hyperstart] # Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017
I0426 09:05:39.894127   21372 vm.go:114] [vm ec005602 hyperstart] *filter
I0426 09:05:39.894176   21372 vm.go:114] [vm ec005602 hyperstart] :INPUT ACCEPT [0:0]
I0426 09:05:39.894226   21372 vm.go:114] [vm ec005602 hyperstart] :FORWARD ACCEPT [0:0]
I0426 09:05:39.894277   21372 vm.go:114] [vm ec005602 hyperstart] :OUTPUT ACCEPT [0:0]
I0426 09:05:39.894328   21372 vm.go:114] [vm ec005602 hyperstart] COMMIT
I0426 09:05:39.894379   21372 vm.go:114] [vm ec005602 hyperstart] # Completed on Wed Apr 26 09:05:39 2017
I0426 09:05:39.894437   21372 vm.go:114] [vm ec005602 hyperstart] hwclock:
I0426 09:05:39.894532   21372 vm.go:114] [vm ec005602 hyperstart] Cannot access the Hardware Clock via any known method.create directory /tmp/hyper
I0426 09:05:39.894567   21372 vm.go:114] [vm ec005602 hyperstart]
I0426 09:05:39.894620   21372 vm.go:114] [vm ec005602 hyperstart] create directory /tmp/hyper/proc
I0426 09:05:39.894704   21372 vm.go:114] [vm ec005602 hyperstart] hwclock: Use the --debug option to see the details of our search for an access method.
I0426 09:05:39.894798   21372 vm.go:114] [vm ec005602 hyperstart] nic eth0 already exists
I0426 09:05:39.894863   21372 vm.go:114] [vm ec005602 hyperstart] net device eth0
I0426 09:05:39.894926   21372 vm.go:114] [vm ec005602 hyperstart] net device sys path is /sys/class/net/eth0/ifindex
I0426 09:05:39.895004   21372 vm.go:114] [vm ec005602 hyperstart] get ifindex 2
I0426 09:05:39.895070   21372 vm.go:114] [vm ec005602 hyperstart] interface get netamsk 16 255.255.0.0
I0426 09:05:39.895154   21372 vm.go:114] [vm ec005602 hyperstart] Setting interface MTU to 1500
I0426 09:05:39.895358   21372 vm.go:114] [vm ec005602 hyperstart] net device eth0
I0426 09:05:39.895435   21372 vm.go:114] [vm ec005602 hyperstart] net device sys path is /sys/class/net/eth0/ifindex
I0426 09:05:39.895534   21372 vm.go:114] [vm ec005602 hyperstart] get ifindex 2
I0426 09:05:39.895613   21372 vm.go:114] [vm ec005602 hyperstart] net device eth0
I0426 09:05:39.895694   21372 vm.go:114] [vm ec005602 hyperstart] net device sys path is /sys/class/net/eth0/ifindex
I0426 09:05:39.895778   21372 vm.go:114] [vm ec005602 hyperstart] get ifindex 2
I0426 09:05:39.895918   21372 vm.go:114] [vm ec005602 hyperstart] execing iptables-restore
I0426 09:05:39.896688   21372 vm.go:114] [vm ec005602 hyperstart] # Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017
I0426 09:05:39.896810   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `PREROUTING'
I0426 09:05:39.896883   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `INPUT'
I0426 09:05:39.896938   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `OUTPUT'
I0426 09:05:39.896996   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `POSTROUTING'
I0426 09:05:39.897089   21372 vm.go:114] [vm ec005602 hyperstart] # Completed on Wed Apr 26 09:05:39 2017
I0426 09:05:39.897142   21372 vm.go:114] [vm ec005602 hyperstart] # Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017
I0426 09:05:39.897217   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `PREROUTING'
I0426 09:05:39.897270   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `INPUT'
I0426 09:05:39.897338   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `FORWARD'
I0426 09:05:39.897396   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `OUTPUT'
I0426 09:05:39.897453   21372 vm.go:114] [vm ec005602 hyperstart] Flushing chain `POSTROUTING'
I0426 09:05:39.897553   21372 vm.go:114] [vm ec005602 hyperstart] # Completed on Wed Apr 26 09:05:39 2017
I0426 09:05:39.897612   21372 vm.go:114] [vm ec005602 hyperstart] # Generated by iptables-save v1.4.21 on Wed Apr 26 09:05:39 2017
I0426 09:05:39.897756   21372 vm.go:114] [vm ec005602 hyperstart] iptables-restore v1.6.1: iptables-restore: unable to initialize table 'security'
I0426 09:05:39.897796   21372 vm.go:114] [vm ec005602 hyperstart]
I0426 09:05:39.897851   21372 vm.go:114] [vm ec005602 hyperstart] Error occurred at line: 19
I0426 09:05:39.897911   21372 vm.go:114] [vm ec005602 hyperstart] Try `iptables-restore -h' or 'iptables-restore --help' for more information.
I0426 09:05:39.898044   21372 vm.go:114] [vm ec005602 hyperstart] Command iptables-restore exited normally, status 2
I0426 09:05:39.898071   21372 vm.go:114] [vm ec005602 hyperstart] Command iptables-restore exit unexpectedly, status 512
I0426 09:05:39.898101   21372 vm.go:114] [vm ec005602 hyperstart] iptable rules setup failed
I0426 09:05:39.898138   21372 vm.go:114] [vm ec005602 hyperstart] hyper send type 10, len 0
I0426 09:05:39.898276   21372 vm.go:114] [vm ec005602 hyperstart] umount /proc/sys/fs/binfmt_misc
I0426 09:05:39.899968   21372 proxy.go:67] [client #1] connection closed
I0426 09:05:39.904955   21372 proxy.go:67] [client #2] client connected
I0426 09:05:39.905033   21372 proxy.go:77] [client #2] attach(containerId=ec005602a273d67601d0249eef5e1b04641b50f7ec17111f6491d58c096c6831)
I0426 09:05:39.905183   21372 proxy.go:77] [client #2] hyper(cmd=destroypod, data={})
I0426 09:05:39.907819   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/fs/cgroup/perf_event
I0426 09:05:39.919777   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/fs/cgroup/memory
I0426 09:05:39.923809   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/fs/cgroup/freezer
I0426 09:05:39.931757   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/fs/cgroup/devices
I0426 09:05:39.935803   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/fs/cgroup/net_cls
I0426 09:05:39.943803   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/fs/cgroup/cpu,cpuacct
I0426 09:05:39.947780   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/kernel/debug
I0426 09:05:39.957313   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/fs/fuse/connections
I0426 09:05:39.961781   21372 vm.go:114] [vm ec005602 hyperstart] umount /dev/mqueue
I0426 09:05:39.969617   21372 vm.go:114] [vm ec005602 hyperstart] umount /sys/fs/cgroup/systemd
I0426 09:05:39.973622   21372 vm.go:114] [vm ec005602 hyperstart] umount /
I0426 09:05:39.993028   21372 proxy.go:67] [client #2] connection closed
I0426 09:05:39.993150   21372 proxy.go:67] [client #3] client connected
I0426 09:05:39.993242   21372 proxy.go:67] [client #3] bye()
I0426 09:05:39.995655   21372 proxy.go:67] [client #3] connection closed

### PR DESCRIPTION
**Pull request by amshinde**
This has kernel configs enabled for adding support for
security xtable. We were seeing issues with running clear containers
on Centos which has kernel config for security xtable enabled.
This was causing iptables command to fail.

Fixes #850

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

**Comment by devimc** [comment]
lgtm

**Comment by jodh-intel** [comment]
nice.

lgtm

**Comment by jcvenegas** [comment]
lgtm

File: versions.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ docker_ubuntu_version=16.04
 docker_engine_fedora_version=1.12.1
 docker_engine_ubuntu_version=1.12.1-0~xenial
 clear_vm_image_version=14700
-clear_vm_kernel_version=4.9.4-57
+clear_vm_kernel_version=4.9.4-60
```
Comment (gorozco1) [inline versions.txt]: Here says it is 4.9.4 not 4.9.24

Comment (jcvenegas) [inline versions.txt]: good catch

**Comment by jcvenegas** [comment]
Also could  you that fixes #837 in the commit message?

**Comment by gorozco1** [comment]
@amshinde Could you also update the linux config and changes in PR #672

File: versions.txt (hunk 1 above)
Comment (amshinde) [inline versions.txt]: Fixed!

**Comment by chavafg** [comment]
qa-passed

**Comment by chavafg** [comment]
qa-passed

**Comment by sboeuf** [comment]
LGTM

**Pull request merged**